src_cpp/elfgames/go/common/go_state_ext.cc

```diff
@@ -82,47 +82,52 @@ std::string GoStateExt::dumpSgf(const std::string& filename) const {
 
 void GoStateExt::showFinishInfo(FinishReason reason) const {
   Stone player = _state.nextPlayer();
-  std::cout << _state.showBoard() << std::endl;
+  _logger->info("{}\n", _state.showBoard());
   std::string sgf_record = dumpSgf("");
-  std::cout << sgf_record << std::endl;
+  _logger->info("{}\n", sgf_record);
 
-  std::cout << "[" << _game_idx << ":" << _seq
-            << "] Current request: " << curr_request_.info()
-            << ", used_model: ";
+  _logger->info("[{}:{}] Current request: {}, used_model: ",
+    _game_idx,
+    _seq,
+    curr_request_.info());
   for (const auto& i : using_models_) {
-    std::cout << i << ", ";
+    _logger->info("{}, ", i);
   }
-  std::cout << std::endl;
+  _logger->info("\n");
 
   switch (reason) {
     case FR_RESIGN:
-      std::cout << "Player " << player2str(player) << " resigned at "
-                << _state.getPly()
-                << " Resign Thres: " << _resign_check.resign_thres;
+      _logger->info("Player {} resigned at {} Resign Thres: {}",
+        player2str(player),
+        _state.getPly(),
+        _resign_check.resign_thres);
       break;
     case FR_MAX_STEP:
-      std::cout << "Ply: " << _state.getPly()
-                << " exceeds thread_state.Restarting the game";
+      _logger->info("Ply: {} exceeds thread_state.Restarting the game",
```

Comment:
Feel free to fix spelling/grammar/formatting mistakes (e.g. space after period here).